### ISSUE TITLE
word express server fails to start

### ISSUE THREAD

**Issue by greenpdx** [ramsaylanier/WordExpress-Server#2]
I am trying to get VuePress working. to get VuePress working I need to get WordExpress_Server working.  There are quit a few steps that are missing for from a clean install.  
I have install nginx, mysql and wordpress and that is working. I changed the database login to "ubuntu" and password.  When I type npm start it fails with this error
I am sure I did not do something but I have no idea what it is.

module.js:549
    throw err;
    ^

Error: Cannot find module './modules/Postmeta/connectors'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/ubuntu/WordExpress-Server/node_modules/wordexpress-schema/lib/db.js:39:20)

**Comment by mcmillenb** [comment]
I'm having this same issue too. I'm guessing it's somehow related to [this commit in WordExpressSchema](https://github.com/ramsaylanier/WordExpressSchema/commit/70e3a6a910c6551a2efeda7e28d89d3321e895f2), but not exactly sure what the issue is.

**Comment by ramsaylanier** [comment]
@greenpdx @mcmillenb I'm taking a look at this today and I'll get back to you ASAP.

**Comment by ramsaylanier** [comment]
@greenpdx @mcmillenb can you please provide me with node version and OS? A link to a repository would also be appreciated, if possible.

**Comment by mcmillenb** [comment]
@ramsaylanier running node v9.11.0 on Linux kernel 2.6.32-042stab127.2. And the repository would just be this repository. I git cloned it, added my config files, ran `npm install` and then `npm start` and it threw the error.

**Comment by ramsaylanier** [comment]
@mcmillenb Okay I'll try running it from a docker image and see what I can find. I'll get back to you shortly.

**Comment by ramsaylanier** [comment]
@mcmillenb to be clear, you did create the `development.json` file in the config directory, per the instructions, yes?

**Comment by ramsaylanier** [comment]
@greenpdx @mcmillenb Okay - I found the issue. I guess I never rebuilt the `wordexpress-schema` package after I [made this commit](https://github.com/ramsaylanier/WordExpressSchema/commit/70e3a6a910c6551a2efeda7e28d89d3321e895f2). I've since rebuilt it and published version 4.3 of `wordexpress-schema` and have updated the dependency in this repository.

You just need to:
```
$ npm install --save wordexpress-schema
```

I've tested this from the latest ubuntu docker image with Node 9.11.0 and it now works.
